wandb/sdk/wandb_run.py

```diff
@@ -4260,13 +4260,13 @@ def _footer_notify_wandb_core(
         printer: Union["PrinterTerm", "PrinterJupyter"],
     ) -> None:
         """Prints a message advertising the upcoming core release."""
-        if quiet or settings._require_core:
+        if quiet or not settings._require_legacy_service:
             return
 
         printer.display(
-            "The new W&B backend becomes opt-out in version 0.18.0;"
-            ' try it out with `wandb.require("core")`!'
-            " See https://wandb.me/wandb-core for more information.",
+            "The legacy backend is deprecated. In version 0.19.0, `wandb-core` will become "
```

Comment:
Are we sure we want to commit to 0.19 for that purpose? How about "In future versions" instead?

Reply:
yeah i was thinking about it, let's not commit, good point